specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-03-01/examples/virtualMachineScaleSetExamples/VirtualMachineScaleSets_Reapply_MaximumSet_Gen.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
instead of generated example, could we have one real example with real values?

Reply:
Done

Reply:
could we update the name to be VirtualMachineScaleSet_Reapply_MaximumSet_Gen.json? 
without the 's' to be singluar. please also update the other example